sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/StructUtils.java

```diff
@@ -38,6 +39,20 @@
 
 final class StructUtils {
 
+  private static final SpannerIO.Read.ToBeamRowFunction STRUCT_TO_BEAM_ROW_FUNCTION =
+      schema -> (Struct struct) -> structToBeamRow(struct, schema);
+
+  public static SpannerIO.Read.ToBeamRowFunction structToBeamRow() {
+    return STRUCT_TO_BEAM_ROW_FUNCTION;
+  }
+
+  private static final SpannerIO.Read.FromBeamRowFunction STRUCT_FROM_BEAM_ROW_FUNCTION =
```

Comment:
Do we need the same type coersion as below?

Reply:
I'm not sure I understand the question. Are you asking if we need to be converting the code differently that the existing Row->Struct function?

Reply:
Oh, I misread this and thought this was a more generic function, not something from this file specifically. Nevermind - please ignore.